Python/Product/DebuggerHelper/trace.cpp

```diff
@@ -266,7 +266,7 @@ bool StringEquals27(const DebuggerString* debuggerString, const void* pyString)
         }
 
         const wchar_t* data = ReadField<const wchar_t*>(pyString, fieldOffsets.PyUnicodeObject27.str);
-        return memcmp(data, my_data, my_length * 2) == 0;
+        return memcmp(data, my_data, my_length * ((size_t) 2)) == 0;
```

Comment:
![image](https://user-images.githubusercontent.com/3315992/53908630-a263b100-4004-11e9-8f9f-bbd264e62fb6.png)

Reply:
Can you do a `static_cast` here and in other places? There's still a bunch of C casts around, but I've been trying to stick to C++-style for all new code here.

Reply:
Will be updated in next commit.